RELEASE_NOTES.md

```diff
@@ -13,6 +13,7 @@
 * Make a distinction between Order and Trade in the protobuf definitions
 * Introduction of new endpoints to retrieve gridpool trades
 * Addition of new definitions and support for trade state filters and streaming
+* Refactor DeliveryDuration to Accept timedelta
```

Comment:
What about something around this:
```suggestion
* Refactor DeliveryDuration to define a timedelta duration attribute instead of defining the class as Enum
```